resources/scss/styles.scss

```diff
@@ -0,0 +1,651 @@
+  font-weight: bold;
+}
+
+.story-members {
+  font-size: 13px;
+  margin-top: 4px;
+}
+
+.about-intro {
+  display: flex;
+  flex-direction: column;
+  background-image: url('../images/about-bg-01.png');
+  background-repeat: no-repeat;
+  height: auto;
+  background-color: $lighter;
+}
+
+.about-welcome {
+  text-align: center;
+}
+
+.about-title {
+  text-align: center;
+}
+
+.about-message {
+  border: $light 4px solid;
+  background-color: white;
+}
+
+.about-contact {
+  font-size: 13px;
+  line-height: 150%;
+  text-align: center;
+}
+
+.about-event-paragraph {
+  font-size: 13px;
+  text-align: justify;
+  margin: 0 auto 16px auto;
+}
+
+.about-prev-group {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.about-prev-img {
+  width: 100%;
+  padding: 10px 20px;
+}
+
+.about-partners {
+  background-color: $dark;
+  color: white;
+}
+
+.about-list {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  flex-wrap: wrap;
+}
+
+.about-item {
+  padding: 8px;
+  font-size: 20px;
+  color: $lighter;
+  list-style: none;
+}
+
+footer {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  margin: 50px 0;
+  padding: 20px;
+  max-width: $max-width;
+  width: 100vw;
+  border-top: $light 2px solid;
+}
+
+.footer-img {
+  margin: 16px 50px;
+  background-color: white;
+  padding: 8px;
+}
+
+.footer-text {
+  text-align: center;
+}
+
+//
+// **********************************
+// desktop version
+// **********************************
+//
+@media (min-width: 768px) {
+  .menu {
+    padding: 0;
+  }
+
+  .menu-top {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: flex-end;
+    background-color: $dark;
+    color: white;
+    height: 36px;
+    max-width: $max-width;
+    width: 100vw;
+    margin: 0;
+    padding: 0;
+    padding-right: 100px;
+  }
+
+  .menu-top-list {
+    display: flex;
+    flex-direction: row;
+    margin: 0;
+  }
+
+  .menu-top-item {
+    font-size: 13px;
+    list-style: none;
+    padding: 4px 0 4px 20px;
+
+    a {
+      color: white;
+    }
+  }
+
+  .menu-main {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    max-width: $max-width;
+    width: 100vw;
+    padding-left: 100px;
+  }
+
+  .menu-img {
+    justify-content: flex-start;
+  }
+
+  .menu-button {
+    display: none;
+  }
+
+  .menu-logo {
+    display: flex;
+    background-color: white;
+  }
+
+  .menu-list {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    height: 80px;
+    margin-top: 0;
+    padding-right: 100px;
+  }
+
+  .menu-item {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    padding: 10px 0 10px 50px;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .section {
+    padding: 100px;
+  }
+
```

Comment:
- [ ] Great job, however, to follow the spacing from the design the partner section is taking up a lot of space please fix that according to the design.

### Your Implementation

![image](https://user-images.githubusercontent.com/59416136/163453669-7fbdfa24-d035-4211-9810-0347b53bea42.png)

## Behance Design

![image](https://user-images.githubusercontent.com/59416136/163453738-a5248bdc-ddd8-4fd6-8e07-ec3fa0f759b9.png)